--- 3.4.2.dump
+++ 3.5.0.dump
@@ -275,0 +276,11 @@
+PersistentTypeID 41
+    Name OcclusionPortal
+    Namespace 
+    Module 
+    Base Component
+    DescendantCount 0
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
@@ -792,0 +804,11 @@
+PersistentTypeID 126
+    Name NavMeshLayers
+    Namespace 
+    Module 
+    Base GlobalGameManager
+    DescendantCount 0
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
@@ -1111,0 +1134,11 @@
+PersistentTypeID 158
+    Name WebCamTexture
+    Namespace 
+    Module 
+    Base Texture
+    DescendantCount 0
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
@@ -1144,0 +1178,11 @@
+PersistentTypeID 162
+    Name EditorUserSettings
+    Namespace 
+    Module 
+    Base Object
+    DescendantCount 0
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
@@ -1309,0 +1354,11 @@
+PersistentTypeID 191
+    Name OffMeshLink
+    Namespace 
+    Module 
+    Base Component
+    DescendantCount 0
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
@@ -1332,2 +1387,2 @@
-PersistentTypeID 1001
-    Name DataTemplate
+PersistentTypeID 194
+    Name NavMesh
@@ -1336,0 +1392,88 @@
+    DescendantCount 0
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 195
+    Name NavMeshAgent
+    Namespace 
+    Module 
+    Base Behaviour
+    DescendantCount 0
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 196
+    Name NavMeshSettings
+    Namespace 
+    Module 
+    Base LevelGameManager
+    DescendantCount 0
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 197
+    Name LightProbes
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 0
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 198
+    Name ParticleSystem
+    Namespace 
+    Module 
+    Base Component
+    DescendantCount 0
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 199
+    Name ParticleSystemRenderer
+    Namespace 
+    Module 
+    Base Renderer
+    DescendantCount 0
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 205
+    Name LODGroup
+    Namespace 
+    Module 
+    Base Component
+    DescendantCount 0
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 220
+    Name LightProbeGroup
+    Namespace 
+    Module 
+    Base Component
+    DescendantCount 0
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 1001
+    Name Prefab
+    Namespace 
+    Module 
+    Base Object
--- 3.5.7.dump
+++ 2017.1.0f3.dump
@@ -6 +6 @@
-    DescendantCount 0
+    DescendantCount 262
@@ -17 +17 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -25 +25 @@
-    Namespace 
+    Namespace Unity
@@ -28 +28 @@
-    DescendantCount 0
+    DescendantCount 112
@@ -39 +39 @@
-    DescendantCount 0
+    DescendantCount 5
@@ -50 +50 @@
-    DescendantCount 0
+    DescendantCount 2
@@ -61 +61 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -72 +72 @@
-    DescendantCount 0
+    DescendantCount 25
@@ -83 +83 @@
-    DescendantCount 0
+    DescendantCount 69
@@ -94 +94 @@
-    DescendantCount 0
+    DescendantCount 31
@@ -105 +105 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -116 +116 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -127 +127 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -138 +138 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -144,2 +144,2 @@
-PersistentTypeID 17
-    Name Pipeline
+PersistentTypeID 18
+    Name EditorExtension
@@ -148,3 +148,3 @@
-    Base Component
-    DescendantCount 0
-    IsAbstract False
+    Base Object
+    DescendantCount 215
+    IsAbstract True
@@ -155,2 +155,2 @@
-PersistentTypeID 18
-    Name EditorExtension
+PersistentTypeID 19
+    Name Physics2DSettings
@@ -159,3 +159,3 @@
-    Base Object
-    DescendantCount 0
-    IsAbstract True
+    Base GlobalGameManager
+    DescendantCount 1
+    IsAbstract False
@@ -171 +171 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -173 +173 @@
-    IsSealed False
+    IsSealed True
@@ -182 +182 @@
-    DescendantCount 0
+    DescendantCount 2
@@ -193 +193 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -195 +195 @@
-    IsSealed False
+    IsSealed True
@@ -204 +204 @@
-    DescendantCount 0
+    DescendantCount 10
@@ -215 +215 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -226 +226 @@
-    DescendantCount 0
+    DescendantCount 14
@@ -237 +237 @@
-    DescendantCount 0
+    DescendantCount 2
@@ -244 +244 @@
-    Name Scene
+    Name OcclusionCullingSettings
@@ -248 +248 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -255 +255 @@
-    Name RenderManager
+    Name GraphicsSettings
@@ -259 +259 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -270 +270 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -281 +281 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -292 +292 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -303 +303 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -314 +314 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -324,2 +324,2 @@
-    Base TextAsset
-    DescendantCount 0
+    Base NamedObject
+    DescendantCount 1
@@ -336 +336 @@
-    DescendantCount 0
+    DescendantCount 4
@@ -342,2 +342,2 @@
-PersistentTypeID 52
-    Name NotificationManager
+PersistentTypeID 50
+    Name Rigidbody2D
@@ -346,2 +346,2 @@
-    Base GlobalGameManager
-    DescendantCount 0
+    Base Component
+    DescendantCount 1
@@ -348,0 +349,11 @@
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 53
+    Name Collider2D
+    Namespace 
+    Module 
+    Base Behaviour
+    DescendantCount 7
+    IsAbstract True
@@ -358 +369 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -360 +371 @@
-    IsSealed False
+    IsSealed True
@@ -369 +380 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -380 +391 @@
-    DescendantCount 0
+    DescendantCount 8
@@ -388 +399 @@
-    Namespace 
+    Namespace Unity
@@ -391 +402 @@
-    DescendantCount 0
+    DescendantCount 6
@@ -396,0 +408,11 @@
+PersistentTypeID 58
+    Name CircleCollider2D
+    Namespace 
+    Module 
+    Base Collider2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
@@ -399 +421 @@
-    Namespace 
+    Namespace Unity
@@ -402 +424,34 @@
-    DescendantCount 0
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 60
+    Name PolygonCollider2D
+    Namespace 
+    Module 
+    Base Collider2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 61
+    Name BoxCollider2D
+    Namespace 
+    Module 
+    Base Collider2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 62
+    Name PhysicsMaterial2D
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
@@ -413 +468 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -415 +470 @@
-    IsSealed False
+    IsSealed True
@@ -424 +479,12 @@
-    DescendantCount 0
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 66
+    Name CompositeCollider2D
+    Namespace 
+    Module 
+    Base Collider2D
+    DescendantCount 1
@@ -430,2 +496,2 @@
-PersistentTypeID 71
-    Name AnimationManager
+PersistentTypeID 68
+    Name EdgeCollider2D
@@ -434,2 +500,24 @@
-    Base GlobalGameManager
-    DescendantCount 0
+    Base Collider2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 70
+    Name CapsuleCollider2D
+    Namespace 
+    Module 
+    Base Collider2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 72
+    Name ComputeShader
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
@@ -445,2 +533,2 @@
-    Base NamedObject
-    DescendantCount 0
+    Base Motion
+    DescendantCount 2
@@ -457 +545 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -468 +556 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -479 +567 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -481 +569 @@
-    IsSealed False
+    IsSealed True
@@ -490 +578 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -501 +589 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -511,2 +599,2 @@
-    Base NamedObject
-    DescendantCount 0
+    Base SampleClip
+    DescendantCount 1
@@ -523 +611,12 @@
-    DescendantCount 0
+    DescendantCount 2
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 86
+    Name CustomRenderTexture
+    Namespace 
+    Module 
+    Base RenderTexture
+    DescendantCount 1
@@ -534 +633 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -545 +644 @@
-    DescendantCount 0
+    DescendantCount 3
@@ -556 +655,23 @@
-    DescendantCount 0
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 90
+    Name Avatar
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 91
+    Name AnimatorController
+    Namespace 
+    Module 
+    Base RuntimeAnimatorController
+    DescendantCount 1
@@ -567 +688 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -572,0 +694,11 @@
+PersistentTypeID 93
+    Name RuntimeAnimatorController
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 3
+    IsAbstract True
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
@@ -578 +710,12 @@
-    DescendantCount 0
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 95
+    Name Animator
+    Namespace 
+    Module 
+    Base Behaviour
+    DescendantCount 1
@@ -589 +732 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -600 +743 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -608 +751 @@
-    Namespace 
+    Namespace TextRenderingPrivate
@@ -611 +754 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -622 +765 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -633 +776 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -635 +778 @@
-    IsSealed False
+    IsSealed True
@@ -644 +787 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -655 +798 @@
-    DescendantCount 0
+    DescendantCount 2
@@ -659 +802 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -666 +809 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -668 +811 @@
-    IsSealed False
+    IsSealed True
@@ -677 +820 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -679 +822 @@
-    IsSealed False
+    IsSealed True
@@ -688 +831 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -699 +842 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -709,2 +852,2 @@
-    Base Texture2D
-    DescendantCount 0
+    Base Texture
+    DescendantCount 1
@@ -721 +864 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -725 +868 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -732 +875 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -743 +886 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -754 +897 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -765 +908 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -776 +919 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -787 +930 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -798 +941 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -805 +948 @@
-    Name NavMeshLayers
+    Name NavMeshProjectSettings
@@ -809,12 +952 @@
-    DescendantCount 0
-    IsAbstract False
-    IsSealed False
-    IsStripped False
-    IsEditorOnly False
-
-PersistentTypeID 127
-    Name HaloManager
-    Namespace 
-    Module 
-    Base LevelGameManager
-    DescendantCount 0
+    DescendantCount 1
@@ -828 +960 @@
-    Namespace 
+    Namespace TextRendering
@@ -831 +963 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -842 +974 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -853 +985 @@
-    DescendantCount 0
+    DescendantCount 101
@@ -864 +996 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -872 +1004 @@
-    Namespace 
+    Namespace TextRenderingPrivate
@@ -875 +1007 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -886 +1018 @@
-    DescendantCount 0
+    DescendantCount 3
@@ -897 +1029 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -908 +1040 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -910 +1042 @@
-    IsSealed False
+    IsSealed True
@@ -919 +1051 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -921 +1053 @@
-    IsSealed False
+    IsSealed True
@@ -930 +1062 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -938 +1070 @@
-    Namespace 
+    Namespace Unity
@@ -941,12 +1073 @@
-    DescendantCount 0
-    IsAbstract False
-    IsSealed False
-    IsStripped False
-    IsEditorOnly False
-
-PersistentTypeID 140
-    Name RaycastCollider
-    Namespace 
-    Module 
-    Base Collider
-    DescendantCount 0
+    DescendantCount 1
@@ -954 +1075 @@
-    IsSealed False
+    IsSealed True
@@ -963 +1084 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -974 +1095 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -985 +1106 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -987 +1108 @@
-    IsSealed False
+    IsSealed True
@@ -993 +1114 @@
-    Namespace 
+    Namespace Unity
@@ -996 +1117 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -998 +1119 @@
-    IsSealed False
+    IsSealed True
@@ -1004 +1125 @@
-    Namespace 
+    Namespace Unity
@@ -1007 +1128 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1009 +1130 @@
-    IsSealed False
+    IsSealed True
@@ -1018 +1139 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1020 +1141 @@
-    IsSealed False
+    IsSealed True
@@ -1029 +1150 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1040 +1161 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1051 +1172 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1062 +1183 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1072,2 +1193,2 @@
-    Base Texture
-    DescendantCount 0
+    Base BaseVideoTexture
+    DescendantCount 1
@@ -1081 +1202 @@
-    Namespace 
+    Namespace Unity
@@ -1084 +1205 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1086 +1207 @@
-    IsSealed False
+    IsSealed True
@@ -1095 +1216 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1106 +1227 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1117 +1238 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1128 +1249 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1138,2 +1259,2 @@
-    Base Texture
-    DescendantCount 0
+    Base BaseVideoTexture
+    DescendantCount 1
@@ -1150 +1271 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1154,23 +1275 @@
-    IsEditorOnly False
-
-PersistentTypeID 160
-    Name InteractiveCloth
-    Namespace 
-    Module 
-    Base Cloth
-    DescendantCount 0
-    IsAbstract False
-    IsSealed False
-    IsStripped False
-    IsEditorOnly False
-
-PersistentTypeID 161
-    Name ClothRenderer
-    Namespace 
-    Module 
-    Base Renderer
-    DescendantCount 0
-    IsAbstract False
-    IsSealed False
-    IsStripped False
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1183 +1282 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1187,12 +1286 @@
-    IsEditorOnly False
-
-PersistentTypeID 163
-    Name SkinnedCloth
-    Namespace 
-    Module 
-    Base Cloth
-    DescendantCount 0
-    IsAbstract False
-    IsSealed False
-    IsStripped False
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1205 +1293 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1216 +1304 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1227 +1315 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1238 +1326 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1249 +1337 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1260 +1348 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1271 +1359,12 @@
-    DescendantCount 0
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 171
+    Name SparseTexture
+    Namespace 
+    Module 
+    Base Texture
+    DescendantCount 1
@@ -1282 +1381 @@
-    DescendantCount 0
+    DescendantCount 3
@@ -1293 +1392 @@
-    DescendantCount 0
+    DescendantCount 7
@@ -1304 +1403 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1312 +1411 @@
-    Namespace 
+    Namespace Unity
@@ -1315,3 +1414,3 @@
-    DescendantCount 0
-    IsAbstract True
-    IsSealed False
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
@@ -1326 +1425 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1337 +1436 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1348 +1447,23 @@
-    DescendantCount 0
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 187
+    Name Texture2DArray
+    Namespace 
+    Module 
+    Base Texture
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 188
+    Name CubemapArray
+    Namespace 
+    Module 
+    Base Texture
+    DescendantCount 1
@@ -1358,2 +1479,2 @@
-    Base Component
-    DescendantCount 0
+    Base Behaviour
+    DescendantCount 1
@@ -1370 +1491 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1381 +1502,23 @@
-    DescendantCount 0
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 195
+    Name NavMeshAgent
+    Namespace 
+    Module 
+    Base Behaviour
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 196
+    Name NavMeshSettings
+    Namespace 
+    Module 
+    Base LevelGameManager
+    DescendantCount 1
@@ -1387,2 +1530,24 @@
-PersistentTypeID 194
-    Name NavMesh
+PersistentTypeID 198
+    Name ParticleSystem
+    Namespace 
+    Module 
+    Base Component
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 199
+    Name ParticleSystemRenderer
+    Namespace 
+    Module 
+    Base Renderer
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 200
+    Name ShaderVariantCollection
@@ -1392 +1557 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1398,2 +1563,35 @@
-PersistentTypeID 195
-    Name NavMeshAgent
+PersistentTypeID 205
+    Name LODGroup
+    Namespace 
+    Module 
+    Base Component
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 206
+    Name BlendTree
+    Namespace 
+    Module 
+    Base Motion
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 207
+    Name Motion
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 4
+    IsAbstract True
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 208
+    Name NavMeshObstacle
@@ -1403 +1601 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1409,2 +1607,2 @@
-PersistentTypeID 196
-    Name NavMeshSettings
+PersistentTypeID 210
+    Name SortingGroup
@@ -1413,2 +1611,24 @@
-    Base LevelGameManager
-    DescendantCount 0
+    Base Behaviour
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 212
+    Name SpriteRenderer
+    Namespace 
+    Module 
+    Base Renderer
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 213
+    Name Sprite
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
@@ -1420,2 +1640,2 @@
-PersistentTypeID 197
-    Name LightProbes
+PersistentTypeID 214
+    Name CachedSpriteAtlas
@@ -1425 +1645 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1428,0 +1649,11 @@
+    IsEditorOnly True
+
+PersistentTypeID 215
+    Name ReflectionProbe
+    Namespace 
+    Module 
+    Base Behaviour
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
@@ -1431,2 +1662,2 @@
-PersistentTypeID 198
-    Name ParticleSystem
+PersistentTypeID 218
+    Name Terrain
@@ -1435,2 +1666,2 @@
-    Base Component
-    DescendantCount 0
+    Base Behaviour
+    DescendantCount 1
@@ -1442,2 +1673,2 @@
-PersistentTypeID 199
-    Name ParticleSystemRenderer
+PersistentTypeID 220
+    Name LightProbeGroup
@@ -1446,2 +1677,2 @@
-    Base Renderer
-    DescendantCount 0
+    Base Behaviour
+    DescendantCount 1
@@ -1453,2 +1684,2 @@
-PersistentTypeID 205
-    Name LODGroup
+PersistentTypeID 221
+    Name AnimatorOverrideController
@@ -1456,0 +1688,11 @@
+    Base RuntimeAnimatorController
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 222
+    Name CanvasRenderer
+    Namespace UI
+    Module 
@@ -1458 +1700,67 @@
-    DescendantCount 0
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 223
+    Name Canvas
+    Namespace UI
+    Module 
+    Base Behaviour
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 224
+    Name RectTransform
+    Namespace UI
+    Module 
+    Base Transform
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 225
+    Name CanvasGroup
+    Namespace UI
+    Module 
+    Base Behaviour
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 226
+    Name BillboardAsset
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 227
+    Name BillboardRenderer
+    Namespace 
+    Module 
+    Base Renderer
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 228
+    Name SpeedTreeWindAsset
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
@@ -1464,2 +1772,497 @@
-PersistentTypeID 220
-    Name LightProbeGroup
+PersistentTypeID 229
+    Name AnchoredJoint2D
+    Namespace 
+    Module 
+    Base Joint2D
+    DescendantCount 8
+    IsAbstract True
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 230
+    Name Joint2D
+    Namespace 
+    Module 
+    Base Behaviour
+    DescendantCount 11
+    IsAbstract True
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 231
+    Name SpringJoint2D
+    Namespace 
+    Module 
+    Base AnchoredJoint2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 232
+    Name DistanceJoint2D
+    Namespace 
+    Module 
+    Base AnchoredJoint2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 233
+    Name HingeJoint2D
+    Namespace 
+    Module 
+    Base AnchoredJoint2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 234
+    Name SliderJoint2D
+    Namespace 
+    Module 
+    Base AnchoredJoint2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 235
+    Name WheelJoint2D
+    Namespace 
+    Module 
+    Base AnchoredJoint2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 236
+    Name ClusterInputManager
+    Namespace 
+    Module 
+    Base GlobalGameManager
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 237
+    Name BaseVideoTexture
+    Namespace 
+    Module 
+    Base Texture
+    DescendantCount 3
+    IsAbstract True
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 238
+    Name NavMeshData
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 240
+    Name AudioMixer
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 2
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 241
+    Name AudioMixerController
+    Namespace 
+    Module 
+    Base AudioMixer
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 243
+    Name AudioMixerGroupController
+    Namespace 
+    Module 
+    Base AudioMixerGroup
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 244
+    Name AudioMixerEffectController
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 245
+    Name AudioMixerSnapshotController
+    Namespace 
+    Module 
+    Base AudioMixerSnapshot
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 246
+    Name PhysicsUpdateBehaviour2D
+    Namespace 
+    Module 
+    Base Behaviour
+    DescendantCount 2
+    IsAbstract True
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 247
+    Name ConstantForce2D
+    Namespace 
+    Module 
+    Base PhysicsUpdateBehaviour2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 248
+    Name Effector2D
+    Namespace 
+    Module 
+    Base Behaviour
+    DescendantCount 6
+    IsAbstract True
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 249
+    Name AreaEffector2D
+    Namespace 
+    Module 
+    Base Effector2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 250
+    Name PointEffector2D
+    Namespace 
+    Module 
+    Base Effector2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 251
+    Name PlatformEffector2D
+    Namespace 
+    Module 
+    Base Effector2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 252
+    Name SurfaceEffector2D
+    Namespace 
+    Module 
+    Base Effector2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 253
+    Name BuoyancyEffector2D
+    Namespace 
+    Module 
+    Base Effector2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 254
+    Name RelativeJoint2D
+    Namespace 
+    Module 
+    Base Joint2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 255
+    Name FixedJoint2D
+    Namespace 
+    Module 
+    Base AnchoredJoint2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 256
+    Name FrictionJoint2D
+    Namespace 
+    Module 
+    Base AnchoredJoint2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 257
+    Name TargetJoint2D
+    Namespace 
+    Module 
+    Base Joint2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 258
+    Name LightProbes
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 259
+    Name LightProbeProxyVolume
+    Namespace 
+    Module 
+    Base Behaviour
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 271
+    Name SampleClip
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 2
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 272
+    Name AudioMixerSnapshot
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 2
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 273
+    Name AudioMixerGroup
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 2
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 280
+    Name NScreenBridge
+    Namespace 
+    Module 
+    Base Object
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 290
+    Name AssetBundleManifest
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 300
+    Name RuntimeInitializeOnLoadManager
+    Namespace 
+    Module 
+    Base GlobalGameManager
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 301
+    Name CloudWebServicesManager
+    Namespace 
+    Module 
+    Base GlobalGameManager
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 303
+    Name UnityAnalyticsManager
+    Namespace 
+    Module 
+    Base GlobalGameManager
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 304
+    Name CrashReportManager
+    Namespace 
+    Module 
+    Base GlobalGameManager
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 305
+    Name PerformanceReportingManager
+    Namespace 
+    Module 
+    Base GlobalGameManager
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 310
+    Name UnityConnectSettings
+    Namespace 
+    Module 
+    Base GlobalGameManager
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 319
+    Name AvatarMask
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 320
+    Name PlayableDirector
+    Namespace 
+    Module 
+    Base Behaviour
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 328
+    Name VideoPlayer
+    Namespace 
+    Module 
+    Base Behaviour
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 329
+    Name VideoClip
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 331
+    Name SpriteMask
+    Namespace 
+    Module 
+    Base Renderer
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 362
+    Name WorldAnchor
@@ -1469 +2272,12 @@
-    DescendantCount 0
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 363
+    Name OcclusionCullingData
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
@@ -1480 +2294 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1484 +2298 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1491 +2305 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1495 +2309 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1502 +2316 @@
-    DescendantCount 0
+    DescendantCount 24
@@ -1506 +2320 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1509 +2323 @@
-    Name AssetDatabase
+    Name AssetDatabaseV1
@@ -1513 +2327 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1517 +2331 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1524 +2338 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1528 +2342 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1535 +2349 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1539 +2353 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1546 +2360 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1550 +2364 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1552,2 +2366,2 @@
-PersistentTypeID 1020
-    Name AudioImporter
+PersistentTypeID 1008
+    Name ComputeShaderImporter
@@ -1557 +2371 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1561 +2375 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1563,2 +2377,2 @@
-PersistentTypeID 1026
-    Name HierarchyState
+PersistentTypeID 1020
+    Name AudioImporter
@@ -1567,2 +2381,2 @@
-    Base Object
-    DescendantCount 0
+    Base AssetImporter
+    DescendantCount 1
@@ -1572 +2386 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1574,2 +2388,2 @@
-PersistentTypeID 1027
-    Name GUIDSerializer
+PersistentTypeID 1026
+    Name HierarchyState
@@ -1579 +2393 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1583 +2397 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1590 +2404 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1594 +2408 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1601 +2415 @@
-    DescendantCount 0
+    DescendantCount 2
@@ -1605 +2419 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1612 +2426 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1616 +2430 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1623 +2437 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1627 +2441 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1629,2 +2443,2 @@
-PersistentTypeID 1034
-    Name NativeFormatImporter
+PersistentTypeID 1032
+    Name SceneAsset
@@ -1633,2 +2447,2 @@
-    Base AssetImporter
-    DescendantCount 0
+    Base DefaultAsset
+    DescendantCount 1
@@ -1638 +2452 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1640,2 +2454,2 @@
-PersistentTypeID 1035
-    Name MonoImporter
+PersistentTypeID 1034
+    Name NativeFormatImporter
@@ -1645 +2459 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1649 +2463 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1651,2 +2465,2 @@
-PersistentTypeID 1037
-    Name AssetServerCache
+PersistentTypeID 1035
+    Name MonoImporter
@@ -1655,2 +2469,2 @@
-    Base Object
-    DescendantCount 0
+    Base AssetImporter
+    DescendantCount 1
@@ -1660 +2474 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1667 +2481 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1671 +2485 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1678 +2492 @@
-    DescendantCount 0
+    DescendantCount 4
@@ -1682 +2496 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1689 +2503 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1693 +2507 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1700 +2514 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1704 +2518 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1711 +2525 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1715 +2529 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1722 +2536 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1726,12 +2540 @@
-    IsEditorOnly False
-
-PersistentTypeID 1046
-    Name DDSImporter
-    Namespace 
-    Module 
-    Base AssetImporter
-    DescendantCount 0
-    IsAbstract False
-    IsSealed False
-    IsStripped False
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1744 +2547 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1748 +2551 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1755 +2558 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1759 +2562 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1762 +2565 @@
-    Name MonoAssemblyImporter
+    Name PluginImporter
@@ -1766 +2569 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1770 +2573 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1777 +2580 @@
-    DescendantCount 0
+    DescendantCount 1
@@ -1781 +2584 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -1783,2 +2586,2 @@
-PersistentTypeID 1052
-    Name PVRImporter
+PersistentTypeID 1055
+    Name IHVImageFormatImporter
@@ -1788 +2591,56 @@
-    DescendantCount 0
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 1101
+    Name AnimatorStateTransition
+    Namespace 
+    Module 
+    Base AnimatorTransitionBase
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 1102
+    Name AnimatorState
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 1105
+    Name HumanTemplate
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 1107
+    Name AnimatorStateMachine
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 1108
+    Name PreviewAnimationClip
+    Namespace 
+    Module 
+    Base AnimationClip
+    DescendantCount 1
@@ -1793,0 +2652,33 @@
+PersistentTypeID 1109
+    Name AnimatorTransition
+    Namespace 
+    Module 
+    Base AnimatorTransitionBase
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 1110
+    Name SpeedTreeImporter
+    Namespace 
+    Module 
+    Base AssetImporter
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 1111
+    Name AnimatorTransitionBase
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 3
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
@@ -1799 +2690,320 @@
-    DescendantCount 0
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 1113
+    Name LightmapParameters
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 1120
+    Name LightingDataAsset
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 1124
+    Name SketchUpImporter
+    Namespace 
+    Module 
+    Base ModelImporter
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 1125
+    Name BuildReport
+    Namespace BuildReporting
+    Module 
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 1126
+    Name PackedAssets
+    Namespace BuildReporting
+    Module 
+    Base Object
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 1127
+    Name VideoClipImporter
+    Namespace 
+    Module 
+    Base AssetImporter
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 1128
+    Name StyleSheetImporter
+    Namespace 
+    Module 
+    Base AssetImporter
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 100000
+    Name int
+    Namespace 
+    Module 
+    Base 
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 100001
+    Name bool
+    Namespace 
+    Module 
+    Base 
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 100002
+    Name float
+    Namespace 
+    Module 
+    Base 
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 100003
+    Name MonoObject
+    Namespace 
+    Module 
+    Base 
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 100004
+    Name Collision
+    Namespace 
+    Module 
+    Base 
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 100005
+    Name Vector3f
+    Namespace 
+    Module 
+    Base 
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 100006
+    Name RootMotionData
+    Namespace 
+    Module 
+    Base 
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 100007
+    Name Collision2D
+    Namespace 
+    Module 
+    Base 
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 100008
+    Name AudioMixerLiveUpdateFloat
+    Namespace 
+    Module 
+    Base 
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 100009
+    Name AudioMixerLiveUpdateBool
+    Namespace 
+    Module 
+    Base 
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 100010
+    Name Polygon2D
+    Namespace 
+    Module 
+    Base 
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 100011
+    Name void
+    Namespace 
+    Module 
+    Base 
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 638013454
+    Name SpriteAtlasDatabase
+    Namespace 
+    Module 
+    Base Object
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 644342135
+    Name CachedSpriteAtlasRuntimeData
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 668709126
+    Name BuiltAssetBundleInfoSet
+    Namespace BuildReporting
+    Module 
+    Base Object
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 687078895
+    Name SpriteAtlas
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 1152215463
+    Name AssemblyJsonAsset
+    Namespace 
+    Module 
+    Base TextAsset
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 1268269756
+    Name GameObjectRecorder
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 1325145578
+    Name LightingDataAssetParent
+    Namespace 
+    Module 
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 1480428607
+    Name LowerResBlitTexture
+    Namespace 
+    Module 
+    Base Texture
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 1766753193
+    Name AssemblyJsonImporter
+    Namespace 
+    Module 
+    Base AssetImporter
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 2089858483
+    Name ScriptedImporter
+    Namespace 
+    Module 
+    Base AssetImporter
+    DescendantCount 1
--- 2017.1.5f1.dump
+++ 2017.2.0f3.dump
@@ -6 +6 @@
-    DescendantCount 262
+    DescendantCount 266
@@ -28 +28 @@
-    DescendantCount 112
+    DescendantCount 117
@@ -83 +83 @@
-    DescendantCount 69
+    DescendantCount 73
@@ -149 +149 @@
-    DescendantCount 215
+    DescendantCount 217
@@ -204 +204 @@
-    DescendantCount 10
+    DescendantCount 11
@@ -336 +336 @@
-    DescendantCount 4
+    DescendantCount 3
@@ -358 +358 @@
-    DescendantCount 7
+    DescendantCount 8
@@ -985 +985 @@
-    DescendantCount 101
+    DescendantCount 98
@@ -2316 +2316 @@
-    DescendantCount 24
+    DescendantCount 22
@@ -2762,11 +2761,0 @@
-PersistentTypeID 1128
-    Name StyleSheetImporter
-    Namespace 
-    Module 
-    Base AssetImporter
-    DescendantCount 1
-    IsAbstract False
-    IsSealed False
-    IsStripped False
-    IsEditorOnly False
-
@@ -2904,0 +2894,33 @@
+PersistentTypeID 19719996
+    Name TilemapCollider2D
+    Namespace 
+    Module 
+    Base Collider2D
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 156049354
+    Name Grid
+    Namespace 
+    Module 
+    Base GridLayout
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 483693784
+    Name TilemapRenderer
+    Namespace 
+    Module 
+    Base Renderer
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
@@ -2949,11 +2970,0 @@
-PersistentTypeID 1152215463
-    Name AssemblyJsonAsset
-    Namespace 
-    Module 
-    Base TextAsset
-    DescendantCount 1
-    IsAbstract False
-    IsSealed False
-    IsStripped False
-    IsEditorOnly True
-
@@ -2993,2 +3004,2 @@
-PersistentTypeID 1766753193
-    Name AssemblyJsonImporter
+PersistentTypeID 1571458007
+    Name RenderPassAttachment
@@ -2997 +3008 @@
-    Base AssetImporter
+    Base Object
@@ -3002 +3013,23 @@
-    IsEditorOnly True
+    IsEditorOnly False
+
+PersistentTypeID 1742807556
+    Name GridLayout
+    Namespace 
+    Module 
+    Base Behaviour
+    DescendantCount 3
+    IsAbstract True
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 1839735485
+    Name Tilemap
+    Namespace 
+    Module 
+    Base GridLayout
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
@@ -3013,0 +3047,11 @@
+
+PersistentTypeID 2126867596
+    Name TilemapEditorUserSettings
+    Namespace 
+    Module 
+    Base Object
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
--- 2017.2.5f1.dump
+++ 2017.3.0f3.dump
@@ -4 +4 @@
-    Module 
+    Module Core
@@ -6 +6 @@
-    DescendantCount 266
+    DescendantCount 267
@@ -15 +15 @@
-    Module 
+    Module Core
@@ -26 +26 @@
-    Module 
+    Module Core
@@ -37 +37 @@
-    Module 
+    Module Core
@@ -48 +48 @@
-    Module 
+    Module Core
@@ -59 +59 @@
-    Module 
+    Module Core
@@ -70 +70 @@
-    Module 
+    Module Core
@@ -81 +81 @@
-    Module 
+    Module Core
@@ -92 +92 @@
-    Module 
+    Module Core
@@ -103 +103 @@
-    Module 
+    Module Audio
@@ -114 +114 @@
-    Module 
+    Module ParticlesLegacy
@@ -125 +125 @@
-    Module 
+    Module Core
@@ -136 +136 @@
-    Module 
+    Module ParticlesLegacy
@@ -147 +147 @@
-    Module 
+    Module Core
@@ -149 +149 @@
-    DescendantCount 217
+    DescendantCount 219
@@ -158 +158 @@
-    Module 
+    Module Physics2D
@@ -169 +169 @@
-    Module 
+    Module Core
@@ -180 +180 @@
-    Module 
+    Module Core
@@ -191 +191 @@
-    Module 
+    Module Core
@@ -202 +202 @@
-    Module 
+    Module Core
@@ -213 +213 @@
-    Module 
+    Module ParticlesLegacy
@@ -224 +224 @@
-    Module 
+    Module Core
@@ -235 +235 @@
-    Module 
+    Module Core
@@ -246 +246 @@
-    Module 
+    Module Umbra
@@ -257 +257 @@
-    Module 
+    Module Core
@@ -268 +268 @@
-    Module 
+    Module Core
@@ -279 +279 @@
-    Module 
+    Module Core
@@ -290 +290 @@
-    Module 
+    Module Core
@@ -301 +301 @@
-    Module 
+    Module Core
@@ -312 +312 @@
-    Module 
+    Module Core
@@ -323 +323 @@
-    Module 
+    Module Core
@@ -334 +334 @@
-    Module 
+    Module Core
@@ -336 +336 @@
-    DescendantCount 3
+    DescendantCount 4
@@ -345 +345 @@
-    Module 
+    Module Physics2D
@@ -356 +356 @@
-    Module 
+    Module Physics2D
@@ -367 +367 @@
-    Module 
+    Module Physics
@@ -378 +378 @@
-    Module 
+    Module Physics
@@ -389 +389 @@
-    Module 
+    Module Physics
@@ -400 +400 @@
-    Module 
+    Module Physics
@@ -411 +411 @@
-    Module 
+    Module Physics2D
@@ -422 +422 @@
-    Module 
+    Module Physics
@@ -433 +433 @@
-    Module 
+    Module Physics2D
@@ -444 +444 @@
-    Module 
+    Module Physics2D
@@ -455 +455 @@
-    Module 
+    Module Physics2D
@@ -466 +466 @@
-    Module 
+    Module Physics
@@ -477 +477 @@
-    Module 
+    Module Physics
@@ -488 +488 @@
-    Module 
+    Module Physics2D
@@ -499 +499 @@
-    Module 
+    Module Physics2D
@@ -510 +510 @@
-    Module 
+    Module Physics2D
@@ -521 +521 @@
-    Module 
+    Module Core
@@ -532 +532 @@
-    Module 
+    Module Animation
@@ -543 +543 @@
-    Module 
+    Module Physics
@@ -554 +554 @@
-    Module 
+    Module ParticlesLegacy
@@ -565 +565 @@
-    Module 
+    Module Core
@@ -576 +576 @@
-    Module 
+    Module Audio
@@ -587 +587 @@
-    Module 
+    Module Audio
@@ -598 +598 @@
-    Module 
+    Module Audio
@@ -609 +609 @@
-    Module 
+    Module Core
@@ -620 +620 @@
-    Module 
+    Module Core
@@ -631 +631 @@
-    Module 
+    Module ParticlesLegacy
@@ -642 +642 @@
-    Module 
+    Module ParticlesLegacy
@@ -653 +653 @@
-    Module 
+    Module Core
@@ -664 +664 @@
-    Module 
+    Module Animation
@@ -675 +675 @@
-    Module 
+    Module Animation
@@ -686 +686 @@
-    Module 
+    Module Core
@@ -697 +697 @@
-    Module 
+    Module Animation
@@ -708 +708 @@
-    Module 
+    Module Core
@@ -719 +719 @@
-    Module 
+    Module Animation
@@ -730 +730 @@
-    Module 
+    Module Core
@@ -741 +741 @@
-    Module 
+    Module Core
@@ -752 +752 @@
-    Module 
+    Module TextRendering
@@ -763 +763 @@
-    Module 
+    Module Core
@@ -774 +774 @@
-    Module 
+    Module Core
@@ -785 +785 @@
-    Module 
+    Module Core
@@ -796 +796 @@
-    Module 
+    Module Animation
@@ -807 +807 @@
-    Module 
+    Module Animation
@@ -818 +818 @@
-    Module 
+    Module Core
@@ -829 +829 @@
-    Module 
+    Module Core
@@ -840 +840 @@
-    Module 
+    Module Core
@@ -851 +851 @@
-    Module 
+    Module Core
@@ -862 +862 @@
-    Module 
+    Module Animation
@@ -873 +873 @@
-    Module 
+    Module Core
@@ -884 +884 @@
-    Module 
+    Module Core
@@ -895 +895 @@
-    Module 
+    Module Core
@@ -906 +906 @@
-    Module 
+    Module Core
@@ -917 +917 @@
-    Module 
+    Module Core
@@ -928 +928 @@
-    Module 
+    Module Core
@@ -939 +939 @@
-    Module 
+    Module Core
@@ -950 +950 @@
-    Module 
+    Module AI
@@ -961 +961 @@
-    Module 
+    Module TextRendering
@@ -972 +972 @@
-    Module 
+    Module Core
@@ -983 +983 @@
-    Module 
+    Module Core
@@ -985 +985 @@
-    DescendantCount 98
+    DescendantCount 100
@@ -994 +994 @@
-    Module 
+    Module Core
@@ -1005 +1005 @@
-    Module 
+    Module TextRendering
@@ -1016 +1016 @@
-    Module 
+    Module Core
@@ -1027 +1027 @@
-    Module 
+    Module Physics
@@ -1038 +1038 @@
-    Module 
+    Module Physics
@@ -1049 +1049 @@
-    Module 
+    Module Physics
@@ -1060 +1060 @@
-    Module 
+    Module Core
@@ -1071 +1071 @@
-    Module 
+    Module Physics
@@ -1082 +1082 @@
-    Module 
+    Module Core
@@ -1093 +1093 @@
-    Module 
+    Module Core
@@ -1104 +1104 @@
-    Module 
+    Module Physics
@@ -1115 +1115 @@
-    Module 
+    Module Physics
@@ -1126 +1126 @@
-    Module 
+    Module Physics
@@ -1137 +1137 @@
-    Module 
+    Module Vehicles
@@ -1148 +1148 @@
-    Module 
+    Module Core
@@ -1159 +1159 @@
-    Module 
+    Module Core
@@ -1170 +1170 @@
-    Module 
+    Module Core
@@ -1181 +1181 @@
-    Module 
+    Module Core
@@ -1192 +1192 @@
-    Module 
+    Module Audio
@@ -1203 +1203 @@
-    Module 
+    Module Physics
@@ -1214 +1214 @@
-    Module 
+    Module TerrainPhysics
@@ -1225 +1225 @@
-    Module 
+    Module Core
@@ -1236 +1236 @@
-    Module 
+    Module Terrain
@@ -1247 +1247 @@
-    Module 
+    Module Core
@@ -1258 +1258 @@
-    Module 
+    Module Audio
@@ -1269 +1269 @@
-    Module 
+    Module Core
@@ -1280 +1280 @@
-    Module 
+    Module Core
@@ -1291 +1291 @@
-    Module 
+    Module Audio
@@ -1302 +1302 @@
-    Module 
+    Module Audio
@@ -1313 +1313 @@
-    Module 
+    Module Audio
@@ -1324 +1324 @@
-    Module 
+    Module Audio
@@ -1335 +1335 @@
-    Module 
+    Module Audio
@@ -1346 +1346 @@
-    Module 
+    Module Audio
@@ -1357 +1357 @@
-    Module 
+    Module Audio
@@ -1368 +1368 @@
-    Module 
+    Module Core
@@ -1379 +1379 @@
-    Module 
+    Module Audio
@@ -1390 +1390 @@
-    Module 
+    Module Audio
@@ -1401 +1401 @@
-    Module 
+    Module Wind
@@ -1412 +1412 @@
-    Module 
+    Module Cloth
@@ -1423 +1423 @@
-    Module 
+    Module Core
@@ -1434 +1434 @@
-    Module 
+    Module Core
@@ -1445 +1445 @@
-    Module 
+    Module Core
@@ -1456 +1456 @@
-    Module 
+    Module Core
@@ -1467 +1467 @@
-    Module 
+    Module Core
@@ -1478 +1478 @@
-    Module 
+    Module AI
@@ -1489 +1489 @@
-    Module 
+    Module Core
@@ -1500 +1500 @@
-    Module 
+    Module Terrain
@@ -1511 +1511 @@
-    Module 
+    Module AI
@@ -1522 +1522 @@
-    Module 
+    Module AI
@@ -1533 +1533 @@
-    Module 
+    Module ParticleSystem
@@ -1544 +1544 @@
-    Module 
+    Module ParticleSystem
@@ -1555 +1555 @@
-    Module 
+    Module Core
@@ -1566 +1566 @@
-    Module 
+    Module Core
@@ -1577 +1577 @@
-    Module 
+    Module Core
@@ -1588 +1588 @@
-    Module 
+    Module Animation
@@ -1599 +1599 @@
-    Module 
+    Module AI
@@ -1610 +1610 @@
-    Module 
+    Module Core
@@ -1621 +1621 @@
-    Module 
+    Module Core
@@ -1632 +1632 @@
-    Module 
+    Module Core
@@ -1643 +1643 @@
-    Module 
+    Module Core
@@ -1654 +1654 @@
-    Module 
+    Module Core
@@ -1665 +1665 @@
-    Module 
+    Module Terrain
@@ -1676 +1676 @@
-    Module 
+    Module Core
@@ -1687 +1687 @@
-    Module 
+    Module Animation
@@ -1698 +1698 @@
-    Module 
+    Module UI
@@ -1709 +1709 @@
-    Module 
+    Module UI
@@ -1720 +1720 @@
-    Module 
+    Module Core
@@ -1731 +1731 @@
-    Module 
+    Module UI
@@ -1742 +1742 @@
-    Module 
+    Module Core
@@ -1753 +1753 @@
-    Module 
+    Module Core
@@ -1764 +1764 @@
-    Module 
+    Module Terrain
@@ -1775 +1775 @@
-    Module 
+    Module Physics2D
@@ -1786 +1786 @@
-    Module 
+    Module Physics2D
@@ -1797 +1797 @@
-    Module 
+    Module Physics2D
@@ -1808 +1808 @@
-    Module 
+    Module Physics2D
@@ -1819 +1819 @@
-    Module 
+    Module Physics2D
@@ -1830 +1830 @@
-    Module 
+    Module Physics2D
@@ -1841 +1841 @@
-    Module 
+    Module Physics2D
@@ -1852 +1852 @@
-    Module 
+    Module ClusterInput
@@ -1863 +1863 @@
-    Module 
+    Module Audio
@@ -1874 +1874 @@
-    Module 
+    Module AI
@@ -1885 +1885 @@
-    Module 
+    Module Audio
@@ -1896 +1896 @@
-    Module 
+    Module Core
@@ -1907 +1907 @@
-    Module 
+    Module Core
@@ -1918 +1918 @@
-    Module 
+    Module Core
@@ -1929 +1929 @@
-    Module 
+    Module Core
@@ -1940 +1940 @@
-    Module 
+    Module Physics2D
@@ -1951 +1951 @@
-    Module 
+    Module Physics2D
@@ -1962 +1962 @@
-    Module 
+    Module Physics2D
@@ -1973 +1973 @@
-    Module 
+    Module Physics2D
@@ -1984 +1984 @@
-    Module 
+    Module Physics2D
@@ -1995 +1995 @@
-    Module 
+    Module Physics2D
@@ -2006 +2006 @@
-    Module 
+    Module Physics2D
@@ -2017 +2017 @@
-    Module 
+    Module Physics2D
@@ -2028 +2028 @@
-    Module 
+    Module Physics2D
@@ -2039 +2039 @@
-    Module 
+    Module Physics2D
@@ -2050 +2050 @@
-    Module 
+    Module Physics2D
@@ -2061 +2061 @@
-    Module 
+    Module Physics2D
@@ -2072 +2072 @@
-    Module 
+    Module Core
@@ -2083 +2083 @@
-    Module 
+    Module Core
@@ -2094 +2094 @@
-    Module 
+    Module Audio
@@ -2105 +2105 @@
-    Module 
+    Module Audio
@@ -2116 +2116 @@
-    Module 
+    Module Audio
@@ -2124,11 +2123,0 @@
-PersistentTypeID 280
-    Name NScreenBridge
-    Namespace 
-    Module 
-    Base Object
-    DescendantCount 1
-    IsAbstract False
-    IsSealed False
-    IsStripped False
-    IsEditorOnly False
-
@@ -2138 +2127 @@
-    Module 
+    Module Core
@@ -2149 +2138 @@
-    Module 
+    Module Core
@@ -2160 +2149 @@
-    Module 
+    Module CloudWebServices
@@ -2171 +2160 @@
-    Module 
+    Module UnityAnalytics
@@ -2182 +2171 @@
-    Module 
+    Module CrashReporting
@@ -2193 +2182 @@
-    Module 
+    Module PerformanceReporting
@@ -2204 +2193 @@
-    Module 
+    Module UnityConnect
@@ -2215 +2204 @@
-    Module 
+    Module Animation
@@ -2226 +2215 @@
-    Module 
+    Module Director
@@ -2237 +2226 @@
-    Module 
+    Module Video
@@ -2248 +2237 @@
-    Module 
+    Module Video
@@ -2259 +2248 @@
-    Module 
+    Module SpriteMask
@@ -2270 +2259 @@
-    Module 
+    Module VR
@@ -2281 +2270 @@
-    Module 
+    Module Core
@@ -2292 +2281 @@
-    Module 
+    Module Core
@@ -2303 +2292 @@
-    Module 
+    Module Core
@@ -2314 +2303 @@
-    Module 
+    Module Core
@@ -2316 +2305 @@
-    DescendantCount 22
+    DescendantCount 23
@@ -2325 +2314 @@
-    Module 
+    Module AssetDatabase
@@ -2336 +2325 @@
-    Module 
+    Module Core
@@ -2347 +2336 @@
-    Module 
+    Module Core
@@ -2358 +2347 @@
-    Module 
+    Module Core
@@ -2369 +2358 @@
-    Module 
+    Module Core
@@ -2380 +2369 @@
-    Module 
+    Module Core
@@ -2391 +2380 @@
-    Module 
+    Module Core
@@ -2402 +2391 @@
-    Module 
+    Module Core
@@ -2413 +2402 @@
-    Module 
+    Module Core
@@ -2424 +2413 @@
-    Module 
+    Module Core
@@ -2435 +2424 @@
-    Module 
+    Module Core
@@ -2446 +2435 @@
-    Module 
+    Module Core
@@ -2457 +2446 @@
-    Module 
+    Module Core
@@ -2468 +2457 @@
-    Module 
+    Module Core
@@ -2479 +2468 @@
-    Module 
+    Module Core
@@ -2490 +2479 @@
-    Module 
+    Module Core
@@ -2501 +2490 @@
-    Module 
+    Module Core
@@ -2512 +2501 @@
-    Module 
+    Module TextRenderingEditor
@@ -2523 +2512 @@
-    Module 
+    Module Core
@@ -2534 +2523 @@
-    Module 
+    Module Core
@@ -2545 +2534 @@
-    Module 
+    Module Core
@@ -2556 +2545 @@
-    Module 
+    Module Core
@@ -2567 +2556 @@
-    Module 
+    Module Core
@@ -2578 +2567 @@
-    Module 
+    Module Core
@@ -2589 +2578 @@
-    Module 
+    Module Core
@@ -2600 +2589 @@
-    Module 
+    Module Core
@@ -2611 +2600 @@
-    Module 
+    Module Core
@@ -2622 +2611 @@
-    Module 
+    Module Core
@@ -2633 +2622 @@
-    Module 
+    Module Core
@@ -2644 +2633 @@
-    Module 
+    Module Animation
@@ -2655 +2644 @@
-    Module 
+    Module Core
@@ -2666 +2655 @@
-    Module 
+    Module Core
@@ -2677 +2666 @@
-    Module 
+    Module Core
@@ -2688 +2677 @@
-    Module 
+    Module Core
@@ -2699 +2688 @@
-    Module 
+    Module Core
@@ -2710 +2699 @@
-    Module 
+    Module Core
@@ -2721 +2710 @@
-    Module 
+    Module SketchUpEditor
@@ -2732 +2721 @@
-    Module 
+    Module BuildReporting
@@ -2743 +2732 @@
-    Module 
+    Module BuildReporting
@@ -2754 +2743 @@
-    Module 
+    Module Core
@@ -2765 +2754 @@
-    Module 
+    Module Core
@@ -2776 +2765 @@
-    Module 
+    Module Core
@@ -2787 +2776 @@
-    Module 
+    Module Core
@@ -2798 +2787 @@
-    Module 
+    Module Core
@@ -2809 +2798 @@
-    Module 
+    Module Core
@@ -2820 +2809 @@
-    Module 
+    Module Core
@@ -2831 +2820 @@
-    Module 
+    Module Core
@@ -2842 +2831 @@
-    Module 
+    Module Core
@@ -2853 +2842 @@
-    Module 
+    Module Core
@@ -2864 +2853 @@
-    Module 
+    Module Core
@@ -2875 +2864 @@
-    Module 
+    Module Core
@@ -2886 +2875 @@
-    Module 
+    Module Core
@@ -2897 +2886 @@
-    Module 
+    Module Tilemap
@@ -2908 +2897 @@
-    Module 
+    Module Grid
@@ -2919 +2908 @@
-    Module 
+    Module Tilemap
@@ -2930 +2919 @@
-    Module 
+    Module Core
@@ -2941 +2930 @@
-    Module 
+    Module Core
@@ -2952 +2941 @@
-    Module 
+    Module BuildReporting
@@ -2963 +2952 @@
-    Module 
+    Module Core
@@ -2970,0 +2960,11 @@
+PersistentTypeID 1152215463
+    Name AssemblyDefinitionAsset
+    Namespace 
+    Module Core
+    Base TextAsset
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
@@ -2974 +2974 @@
-    Module 
+    Module Core
@@ -2985 +2985 @@
-    Module 
+    Module Core
@@ -2996 +2996 @@
-    Module 
+    Module Core
@@ -3007 +3007 @@
-    Module 
+    Module Core
@@ -3018 +3018 @@
-    Module 
+    Module Grid
@@ -3025,0 +3026,11 @@
+PersistentTypeID 1766753193
+    Name AssemblyDefinitionImporter
+    Namespace 
+    Module Core
+    Base AssetImporter
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
@@ -3029 +3040 @@
-    Module 
+    Module Tilemap
@@ -3040 +3051 @@
-    Module 
+    Module Core
@@ -3051 +3062 @@
-    Module 
+    Module TilemapEditor
--- 2017.4.35f1.dump
+++ 2017.4.36f1.dump
@@ -2639 +2639 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -2980 +2980 @@
-    IsEditorOnly False
+    IsEditorOnly True
@@ -3057 +3057 @@
-    IsEditorOnly False
+    IsEditorOnly True
--- 2017.4.40f1.dump
+++ 2018.1.0f2.dump
@@ -6 +6 @@
-    DescendantCount 267
+    DescendantCount 280
@@ -28 +28 @@
-    DescendantCount 117
+    DescendantCount 124
@@ -83 +83 @@
-    DescendantCount 73
+    DescendantCount 79
@@ -149 +149 @@
-    DescendantCount 219
+    DescendantCount 230
@@ -204 +204 @@
-    DescendantCount 11
+    DescendantCount 12
@@ -985 +985 @@
-    DescendantCount 100
+    DescendantCount 104
@@ -1093 +1093 @@
-    Module Core
+    Module AssetBundle
@@ -1269 +1269 @@
-    Module Core
+    Module Editor
@@ -1280 +1280 @@
-    Module Core
+    Module Editor
@@ -1423 +1423 @@
-    Module Core
+    Module Substance
@@ -1434 +1434 @@
-    Module Core
+    Module Substance
@@ -1445 +1445 @@
-    Module Core
+    Module Substance
@@ -1577 +1577 @@
-    Module Core
+    Module Editor
@@ -1896 +1896 @@
-    Module Core
+    Module Editor
@@ -1907 +1907 @@
-    Module Core
+    Module Editor
@@ -1918 +1918 @@
-    Module Core
+    Module Editor
@@ -1929 +1929 @@
-    Module Core
+    Module Editor
@@ -2127 +2127 @@
-    Module Core
+    Module AssetBundle
@@ -2303 +2303 @@
-    Module Core
+    Module Editor
@@ -2336 +2336 @@
-    Module Core
+    Module Editor
@@ -2347 +2347 @@
-    Module Core
+    Module Editor
@@ -2369 +2369 @@
-    Module Core
+    Module Editor
@@ -2402 +2402 @@
-    Module Core
+    Module Editor
@@ -2435 +2435 @@
-    Module Core
+    Module Editor
@@ -2457 +2457 @@
-    Module Core
+    Module Editor
@@ -2479 +2479 @@
-    Module Core
+    Module Editor
@@ -2501 +2501 @@
-    Module TextRenderingEditor
+    Module Editor
@@ -2512 +2512 @@
-    Module Core
+    Module Editor
@@ -2523 +2523 @@
-    Module Core
+    Module Editor
@@ -2556 +2556 @@
-    Module Core
+    Module Editor
@@ -2567 +2567 @@
-    Module Core
+    Module Editor
@@ -2578 +2578 @@
-    Module Core
+    Module Editor
@@ -2589 +2589 @@
-    Module Core
+    Module Editor
@@ -2600 +2600 @@
-    Module Core
+    Module Editor
@@ -2611 +2611 @@
-    Module Core
+    Module Editor
@@ -2622 +2622 @@
-    Module Core
+    Module Editor
@@ -2633 +2633 @@
-    Module Animation
+    Module Core
@@ -2639 +2639 @@
-    IsEditorOnly True
+    IsEditorOnly False
@@ -2644 +2644 @@
-    Module Core
+    Module Editor
@@ -2655 +2655 @@
-    Module Core
+    Module Editor
@@ -2666 +2666 @@
-    Module Core
+    Module Editor
@@ -2677 +2677 @@
-    Module Core
+    Module SubstanceEditor
@@ -2688 +2688 @@
-    Module Core
+    Module Editor
@@ -2699 +2699 @@
-    Module Core
+    Module Editor
@@ -2710 +2710 @@
-    Module SketchUpEditor
+    Module Editor
@@ -2721 +2721 @@
-    Module BuildReporting
+    Module Editor
@@ -2732 +2732 @@
-    Module BuildReporting
+    Module BuildReportingEditor
@@ -2743 +2743 @@
-    Module Core
+    Module Editor
@@ -2754 +2754 @@
-    Module Core
+    Module undefined
@@ -2765 +2765 @@
-    Module Core
+    Module undefined
@@ -2776 +2776 @@
-    Module Core
+    Module undefined
@@ -2787 +2787 @@
-    Module Core
+    Module undefined
@@ -2798 +2798 @@
-    Module Core
+    Module undefined
@@ -2809 +2809 @@
-    Module Core
+    Module undefined
@@ -2820 +2820 @@
-    Module Core
+    Module undefined
@@ -2831 +2831 @@
-    Module Core
+    Module undefined
@@ -2842 +2842 @@
-    Module Core
+    Module undefined
@@ -2853 +2853 @@
-    Module Core
+    Module undefined
@@ -2864 +2864 @@
-    Module Core
+    Module undefined
@@ -2875 +2875 @@
-    Module Core
+    Module undefined
@@ -2893,0 +2894,11 @@
+PersistentTypeID 41386430
+    Name AssetImporterLog
+    Namespace 
+    Module Core
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
@@ -2904,0 +2916,44 @@
+PersistentTypeID 181963792
+    Name Preset
+    Namespace 
+    Module Editor
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 285090594
+    Name IConstraint
+    Namespace 
+    Module Animation
+    Base Behaviour
+    DescendantCount 6
+    IsAbstract True
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 369655926
+    Name AssetImportInProgressProxy
+    Namespace 
+    Module Editor
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 426301858
+    Name EditorProjectAccess
+    Namespace 
+    Module Editor
+    Base NamedObject
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
@@ -2941 +2996 @@
-    Module BuildReporting
+    Module BuildReportingEditor
@@ -2959,0 +3015,22 @@
+PersistentTypeID 877146078
+    Name PlatformModuleSetup
+    Namespace 
+    Module Core
+    Base Object
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
+PersistentTypeID 895512359
+    Name AimConstraint
+    Namespace 
+    Module Animation
+    Base IConstraint
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
@@ -2963 +3040 @@
-    Module Core
+    Module Editor
@@ -2974 +3051 @@
-    Module Core
+    Module Editor
@@ -2980 +3057 @@
-    IsEditorOnly True
+    IsEditorOnly False
@@ -2992,0 +3070,11 @@
+PersistentTypeID 1386491679
+    Name PresetManager
+    Namespace 
+    Module Editor
+    Base Object
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly True
+
@@ -3029 +3117 @@
-    Module Core
+    Module Editor
@@ -3036,0 +3125,44 @@
+PersistentTypeID 1773428102
+    Name ParentConstraint
+    Namespace 
+    Module Animation
+    Base IConstraint
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 1818360608
+    Name PositionConstraint
+    Namespace 
+    Module Animation
+    Base IConstraint
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 1818360609
+    Name RotationConstraint
+    Namespace 
+    Module Animation
+    Base IConstraint
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 1818360610
+    Name ScaleConstraint
+    Namespace 
+    Module Animation
+    Base IConstraint
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
@@ -3047,0 +3180,11 @@
+PersistentTypeID 1971053207
+    Name SpriteShapeRenderer
+    Namespace 
+    Module SpriteShape
+    Base Renderer
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
@@ -3051 +3194 @@
-    Module Core
+    Module Editor
@@ -3057 +3200 @@
-    IsEditorOnly True
+    IsEditorOnly False
@@ -3062 +3205 @@
-    Module TilemapEditor
+    Module Editor
--- 2018.1.9f2.dump
+++ 2018.2.0f2.dump
@@ -6 +6 @@
-    DescendantCount 280
+    DescendantCount 282
@@ -28 +28 @@
-    DescendantCount 124
+    DescendantCount 125
@@ -72 +72 @@
-    DescendantCount 25
+    DescendantCount 24
@@ -83 +83 @@
-    DescendantCount 79
+    DescendantCount 80
@@ -94 +94 @@
-    DescendantCount 31
+    DescendantCount 30
@@ -149 +149 @@
-    DescendantCount 230
+    DescendantCount 233
@@ -985 +985 @@
-    DescendantCount 104
+    DescendantCount 106
@@ -1156,22 +1155,0 @@
-PersistentTypeID 148
-    Name NetworkView
-    Namespace 
-    Module Core
-    Base Behaviour
-    DescendantCount 1
-    IsAbstract False
-    IsSealed False
-    IsStripped False
-    IsEditorOnly False
-
-PersistentTypeID 149
-    Name NetworkManager
-    Namespace 
-    Module Core
-    Base GlobalGameManager
-    DescendantCount 1
-    IsAbstract False
-    IsSealed False
-    IsStripped False
-    IsEditorOnly False
-
@@ -1222,11 +1199,0 @@
-PersistentTypeID 155
-    Name MasterServerInterface
-    Namespace 
-    Module Core
-    Base GlobalGameManager
-    DescendantCount 1
-    IsAbstract False
-    IsSealed False
-    IsStripped False
-    IsEditorOnly False
-
@@ -2305 +2272 @@
-    DescendantCount 23
+    DescendantCount 24
@@ -2932 +2899 @@
-    DescendantCount 6
+    DescendantCount 7
@@ -3036,0 +3004,11 @@
+PersistentTypeID 1027052791
+    Name LocalizationImporter
+    Namespace 
+    Module LocalizationEditor
+    Base AssetImporter
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly True
+
@@ -3047,0 +3026,11 @@
+PersistentTypeID 1183024399
+    Name LookAtConstraint
+    Namespace 
+    Module Animation
+    Base IConstraint
+    DescendantCount 1
+    IsAbstract False
+    IsSealed True
+    IsStripped False
+    IsEditorOnly False
+
@@ -3080,0 +3070,11 @@
+PersistentTypeID 1403656975
+    Name StreamingManager
+    Namespace 
+    Module Streaming
+    Base GlobalGameManager
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
@@ -3091,0 +3092,11 @@
+PersistentTypeID 1542919678
+    Name StreamingController
+    Namespace 
+    Module Streaming
+    Base Behaviour
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
@@ -3184,0 +3196,11 @@
+    DescendantCount 1
+    IsAbstract False
+    IsSealed False
+    IsStripped False
+    IsEditorOnly False
+
+PersistentTypeID 2083778819
+    Name LocalizationAsset
+    Namespace 
+    Module Localization
+    Base NamedObject
