--- 3.4.2
+++ 3.5.0
@@ -26,0 +27 @@
+  "41": "OcclusionPortal",
@@ -73,0 +75 @@
+  "126": "NavMeshLayers",
@@ -102,0 +105 @@
+  "158": "WebCamTexture",
@@ -105,0 +109 @@
+  "162": "EditorUserSettings",
@@ -120,0 +125 @@
+  "191": "OffMeshLink",
@@ -123 +128,9 @@
-  "1001": "DataTemplate",
+  "194": "NavMesh",
+  "195": "NavMeshAgent",
+  "196": "NavMeshSettings",
+  "197": "LightProbes",
+  "198": "ParticleSystem",
+  "199": "ParticleSystemRenderer",
+  "205": "LODGroup",
+  "220": "LightProbeGroup",
+  "1001": "Prefab",
--- 3.5.7
+++ 4.0.0
@@ -24 +24 @@
-  "29": "Scene",
+  "29": "SceneSettings",
@@ -41,0 +42 @@
+  "72": "ComputeShader",
@@ -52,0 +54,2 @@
+  "90": "Avatar",
+  "91": "AnimatorController",
@@ -54,0 +58 @@
+  "95": "Animator",
@@ -134,0 +139,3 @@
+  "206": "BlendTree",
+  "207": "Motion",
+  "208": "NavMeshObstacle",
@@ -142,0 +150,3 @@
+  "1008": "ComputeShaderImporter",
+  "1011": "AvatarBodyMask",
+  "1012": "AvatarSkeletonMask",
@@ -149,0 +160 @@
+  "1032": "SceneAsset",
@@ -164,0 +176,4 @@
+  "1101": "Transition",
+  "1102": "State",
+  "1105": "HumanTemplate",
+  "1107": "StateMachine",
--- 4.0.1
+++ 4.1.0
@@ -33 +32,0 @@
-  "52": "NotificationManager",
@@ -56,0 +56 @@
+  "93": "RuntimeAnimatorController",
@@ -151,2 +151 @@
-  "1011": "AvatarBodyMask",
-  "1012": "AvatarSkeletonMask",
+  "1011": "AvatarMask",
--- 4.1.5
+++ 4.2.0
@@ -25 +25 @@
-  "30": "RenderManager",
+  "30": "GraphicsSettings",
--- 4.2.2
+++ 4.3.0
@@ -16,0 +17 @@
+  "19": "Physics2DSettings",
@@ -32,0 +34,2 @@
+  "50": "Rigidbody2D",
+  "53": "Collider2D",
@@ -36,0 +40 @@
+  "58": "CircleCollider2D",
@@ -37,0 +42,3 @@
+  "60": "PolygonCollider2D",
+  "61": "BoxCollider2D",
+  "62": "PhysicsMaterial2D",
@@ -40 +47,2 @@
-  "71": "AnimationManager",
+  "68": "EdgeCollider2D",
+  "69": "PolygonColliderBase2D",
@@ -141,0 +150,3 @@
+  "212": "SpriteRenderer",
+  "213": "Sprite",
+  "214": "CachedSpriteAtlas",
@@ -142,0 +154,6 @@
+  "221": "AnimatorOverrideController",
+  "230": "Joint2D",
+  "231": "SpringJoint2D",
+  "232": "DistanceJoint2D",
+  "233": "HingeJoint2D",
+  "234": "SliderJoint2D",
--- 4.3.4
+++ 4.5.0
@@ -129,0 +130 @@
+  "171": "SparseTexture",
@@ -154,0 +156,4 @@
+  "222": "UIRenderer",
+  "223": "Canvas",
+  "224": "RectTransform",
+  "229": "AnchoredJoint2D",
@@ -159,0 +165 @@
+  "235": "WheelJoint2D",
--- 4.5.5
+++ 4.6.0
@@ -156 +156 @@
-  "222": "UIRenderer",
+  "222": "CanvasRenderer",
@@ -158,0 +159 @@
+  "225": "CanvasGroup",
--- 4.7.2
+++ 5.0.0f4
@@ -15 +14,0 @@
-  "17": "Pipeline",
@@ -48 +46,0 @@
-  "69": "PolygonColliderBase2D",
@@ -87 +85 @@
-  "126": "NavMeshLayers",
+  "126": "NavMeshAreas",
@@ -100 +97,0 @@
-  "140": "RaycastCollider",
@@ -119,2 +115,0 @@
-  "160": "InteractiveCloth",
-  "161": "ClothRenderer",
@@ -122 +116,0 @@
-  "163": "SkinnedCloth",
@@ -141 +135 @@
-  "194": "NavMesh",
+  "194": "NavMeshObsolete",
@@ -144 +137,0 @@
-  "197": "LightProbes",
@@ -146,0 +140 @@
+  "200": "ShaderVariantCollection",
@@ -153,0 +148,2 @@
+  "215": "ReflectionProbe",
+  "218": "Terrain",
@@ -159,0 +156,3 @@
+  "226": "BillboardAsset",
+  "227": "BillboardRenderer",
+  "228": "SpeedTreeWindAsset",
@@ -166,0 +166,21 @@
+  "237": "BaseVideoTexture",
+  "238": "NavMeshData",
+  "240": "AudioMixer",
+  "241": "AudioMixerController",
+  "243": "AudioMixerGroupController",
+  "244": "AudioMixerEffectController",
+  "245": "AudioMixerSnapshotController",
+  "246": "PhysicsUpdateBehaviour2D",
+  "247": "ConstantForce2D",
+  "248": "Effector2D",
+  "249": "AreaEffector2D",
+  "250": "PointEffector2D",
+  "251": "PlatformEffector2D",
+  "252": "SurfaceEffector2D",
+  "258": "LightProbes",
+  "271": "SampleClip",
+  "272": "AudioMixerSnapshot",
+  "273": "AudioMixerGroup",
+  "280": "NScreenBridge",
+  "290": "AssetBundleManifest",
+  "300": "RuntimeInitializeOnLoadManager",
@@ -196 +216 @@
-  "1050": "MonoAssemblyImporter",
+  "1050": "PluginImporter",
@@ -199,2 +219,4 @@
-  "1101": "Transition",
-  "1102": "State",
+  "1053": "ASTCImporter",
+  "1054": "KTXImporter",
+  "1101": "AnimatorStateTransition",
+  "1102": "AnimatorState",
@@ -202,2 +224,7 @@
-  "1107": "StateMachine",
-  "1112": "SubstanceImporter"
+  "1107": "AnimatorStateMachine",
+  "1109": "AnimatorTransition",
+  "1110": "SpeedTreeImporter",
+  "1111": "AnimatorTransitionBase",
+  "1112": "SubstanceImporter",
+  "1113": "LightmapParameters",
+  "1120": "LightmapSnapshot"
--- 5.0.4f1
+++ 5.1.0f1
@@ -186,0 +187,2 @@
+  "301": "CloudWebServicesManager",
+  "302": "CloudServiceHandlerBehaviour",
@@ -198 +199,0 @@
-  "1027": "GUIDSerializer",
@@ -230 +231,2 @@
-  "1120": "LightmapSnapshot"
+  "1120": "LightmapSnapshot",
+  "1124": "SketchUpImporter"
--- 5.1.5f1
+++ 5.2.0f2
@@ -86 +85,0 @@
-  "127": "HaloManager",
@@ -185,0 +185 @@
+  "292": "UnityAdsSettings",
@@ -188 +188,2 @@
-  "302": "CloudServiceHandlerBehaviour",
+  "303": "UnityAnalyticsManager",
+  "320": "DirectorPlayer",
--- 5.2.5f1
+++ 5.3.0f1
@@ -134 +133,0 @@
-  "194": "NavMeshObsolete",
@@ -164,0 +164 @@
+  "236": "ClusterInputManager",
@@ -178,0 +179,5 @@
+  "253": "BuoyancyEffector2D",
+  "254": "RelativeJoint2D",
+  "255": "FixedJoint2D",
+  "256": "FrictionJoint2D",
+  "257": "TargetJoint2D",
@@ -188,0 +194 @@
+  "310": "UnityConnectSettings",
@@ -232 +238 @@
-  "1120": "LightmapSnapshot",
+  "1120": "LightingDataAsset",
--- 5.3.8f2
+++ 5.4.0b22
@@ -130,0 +131 @@
+  "187": "Texture2DArray",
@@ -184,0 +186 @@
+  "259": "LightProbeProxyVolume",
@@ -239 +241,3 @@
-  "1124": "SketchUpImporter"
+  "1124": "SketchUpImporter",
+  "1125": "BuildReport",
+  "1126": "PackedAssets"
--- 5.4.6f3
+++ 5.5.0f3
@@ -24 +24 @@
-  "29": "SceneSettings",
+  "29": "OcclusionCullingSettings",
@@ -46,0 +47 @@
+  "70": "CapsuleCollider2D",
@@ -85 +86 @@
-  "126": "NavMeshAreas",
+  "126": "NavMeshProjectSettings",
@@ -131,0 +133 @@
+  "188": "CubemapArray",
@@ -192 +194 @@
-  "292": "UnityAdsSettings",
+  "292": "UnityAdsManager",
@@ -195,0 +198 @@
+  "304": "CrashReportManager",
@@ -197,0 +201,2 @@
+  "362": "WorldAnchor",
+  "363": "OcclusionCullingData",
@@ -243 +248,13 @@
-  "1126": "PackedAssets"
+  "1126": "PackedAssets",
+  "100000": "int",
+  "100001": "bool",
+  "100002": "float",
+  "100003": "MonoObject",
+  "100004": "Collision",
+  "100005": "Vector3f",
+  "100006": "RootMotionData",
+  "100007": "Collision2D",
+  "100008": "AudioMixerLiveUpdateFloat",
+  "100009": "AudioMixerLiveUpdateBool",
+  "100010": "Polygon2D",
+  "100011": "void"
--- 5.5.3f1
+++ 5.5.4f1
@@ -248,0 +249 @@
+  "1127": "BuiltAssetBundleInfoSet",
--- 5.5.6f1
+++ 5.6.0b1
@@ -45,0 +46 @@
+  "66": "CompositeCollider2D",
@@ -56,0 +58 @@
+  "86": "CustomRenderTexture",
@@ -145,0 +148 @@
+  "210": "SortingGroup",
@@ -198,0 +202 @@
+  "305": "PerformanceReportingManager",
@@ -200 +204,2 @@
-  "320": "DirectorPlayer",
+  "328": "VideoPlayer",
+  "329": "VideoClip",
@@ -228 +232,0 @@
-  "1046": "DDSImporter",
@@ -233,3 +237 @@
-  "1052": "PVRImporter",
-  "1053": "ASTCImporter",
-  "1054": "KTXImporter",
+  "1055": "IHVImageFormatImporter",
@@ -239,0 +242 @@
+  "1108": "PreviewAnimationClip",
@@ -249 +252 @@
-  "1127": "BuiltAssetBundleInfoSet",
+  "1127": "VideoClipImporter",
--- 5.6.0b2
+++ 5.6.0b3
@@ -203,0 +204 @@
+  "319": "AvatarMask",
@@ -216 +216,0 @@
-  "1011": "AvatarMask",
--- 5.6.0b4
+++ 5.6.0b5
@@ -204,0 +205 @@
+  "327": "VideoClip",
@@ -206 +206,0 @@
-  "329": "VideoClip",
@@ -264 +264,2 @@
-  "100011": "void"
+  "100011": "void",
+  "1480428607": "LowerResBlitTexture"
--- 5.6.0b9
+++ 5.6.0b10
@@ -205 +204,0 @@
-  "327": "VideoClip",
@@ -206,0 +206 @@
+  "329": "VideoClip",
--- 5.6.1f1
+++ 5.6.2f1
@@ -264,0 +265 @@
+  "668709126": "BuiltAssetBundleInfoSet",
--- 5.6.7f1
+++ 2017.1.0b1
@@ -204,0 +205 @@
+  "320": "PlayableDirector",
@@ -206,0 +208 @@
+  "331": "SpriteMask",
@@ -212 +214 @@
-  "1004": "AssetDatabase",
+  "1004": "AssetDatabaseV1",
@@ -226 +227,0 @@
-  "1037": "AssetServerCache",
@@ -252,0 +254 @@
+  "1128": "StyleSheetImporter",
@@ -265,2 +267,7 @@
-  "668709126": "BuiltAssetBundleInfoSet",
-  "1480428607": "LowerResBlitTexture"
+  "638013454": "SpriteAtlasDatabase",
+  "644342135": "CachedSpriteAtlasRuntimeData",
+  "687078895": "SpriteAtlas",
+  "1152215463": "AssemblyJsonAsset",
+  "1268269756": "GameObjectRecorder",
+  "1480428607": "LowerResBlitTexture",
+  "1766753193": "AssemblyJsonImporter"
--- 2017.1.0b1
+++ 2017.1.0b2
@@ -197 +196,0 @@
-  "292": "UnityAdsManager",
@@ -273 +272,2 @@
-  "1766753193": "AssemblyJsonImporter"
+  "1766753193": "AssemblyJsonImporter",
+  "2089858483": "ScriptedImporter"
--- 2017.1.0b3
+++ 2017.1.0b4
@@ -267,0 +268 @@
+  "668709126": "BuiltAssetBundleInfoSet",
--- 2017.1.0b9
+++ 2017.1.0b10
@@ -271,0 +272 @@
+  "1325145578": "LightingDataAssetParent",
--- 2017.1.5f1
+++ 2017.2.0b2
@@ -253 +252,0 @@
-  "1128": "StyleSheetImporter",
@@ -265,0 +265,3 @@
+  "19719996": "TilemapCollider2D",
+  "156049354": "Grid",
+  "483693784": "TilemapRenderer",
@@ -273,0 +276,2 @@
+  "1571458007": "RenderPassAttachment",
+  "1742807556": "GridLayout",
@@ -275 +279,3 @@
-  "2089858483": "ScriptedImporter"
+  "1839735485": "Tilemap",
+  "2089858483": "ScriptedImporter",
+  "2126867596": "TilemapEditorUserSettings"
--- 2017.2.0b10
+++ 2017.2.0b11
@@ -272 +271,0 @@
-  "1152215463": "AssemblyJsonAsset",
@@ -278 +276,0 @@
-  "1766753193": "AssemblyJsonImporter",
--- 2017.2.5f1
+++ 2017.3.0b1
@@ -195 +194,0 @@
-  "280": "NScreenBridge",
@@ -271,0 +271 @@
+  "1152215463": "AssemblyDefinitionAsset",
@@ -276,0 +277 @@
+  "1766753193": "AssemblyDefinitionImporter",
--- 2017.4.40f1
+++ 2018.1.0b2
@@ -264,0 +265 @@
+  "41386430": "AssetImporterLog",
@@ -265,0 +267,4 @@
+  "181963792": "Preset",
+  "285090594": "IConstraint",
+  "369655926": "AssetImportInProgressProxy",
+  "426301858": "EditorProjectAccess",
@@ -270,0 +276,2 @@
+  "877146078": "PlatformModuleSetup",
+  "895512359": "AimConstraint",
@@ -273,0 +281 @@
+  "1386491679": "PresetManager",
@@ -277,0 +286,4 @@
+  "1773428102": "ParentConstraint",
+  "1818360608": "PositionConstraint",
+  "1818360609": "RotationConstraint",
+  "1818360610": "ScaleConstraint",
@@ -278,0 +291 @@
+  "1971053207": "SpriteShapeRenderer",
--- 2018.1.9f2
+++ 2018.2.0b1
@@ -107,2 +106,0 @@
-  "148": "NetworkView",
-  "149": "NetworkManager",
@@ -113 +110,0 @@
-  "155": "MasterServerInterface",
@@ -277,0 +275 @@
+  "1027052791": "LocalizationImporter",
@@ -278,0 +277 @@
+  "1183024399": "LookAtConstraint",
@@ -281,0 +281 @@
+  "1403656975": "StreamingManager",
@@ -282,0 +283 @@
+  "1542919678": "StreamingController",
@@ -291,0 +293 @@
+  "2083778819": "LocalizationAsset",
--- 2018.2.21f1
+++ 2018.3.0b1
@@ -12 +11,0 @@
-  "12": "ParticleAnimator",
@@ -14 +12,0 @@
-  "15": "EllipsoidParticleEmitter",
@@ -21 +18,0 @@
-  "26": "ParticleRenderer",
@@ -52 +48,0 @@
-  "76": "WorldParticleCollider",
@@ -59,2 +54,0 @@
-  "87": "MeshParticleEmitter",
-  "88": "ParticleEmitter",
@@ -194,4 +187,0 @@
-  "301": "CloudWebServicesManager",
-  "303": "UnityAnalyticsManager",
-  "304": "CrashReportManager",
-  "305": "PerformanceReportingManager",
@@ -202,0 +193 @@
+  "330": "ParticleSystemForceField",
@@ -206 +197 @@
-  "1001": "Prefab",
+  "1001": "PrefabInstance",
@@ -262,0 +254 @@
+  "73398921": "VFXRenderer",
@@ -267,0 +260 @@
+  "468431735": "PrefabImporter",
@@ -274,0 +268,2 @@
+  "937362698": "VFXManager",
+  "1001480554": "Prefab",
@@ -291,0 +287 @@
+  "1953259897": "TerrainLayer",
@@ -292,0 +289,4 @@
+  "2058629509": "VisualEffectAsset",
+  "2058629510": "VisualEffectImporter",
+  "2058629511": "VisualEffectResource",
+  "2083052967": "VisualEffect",
--- 2018.4.12f1
+++ 2018.4.13f1
@@ -258,0 +259 @@
+  "382020655": "PluginBuildInfo",
@@ -262,0 +264 @@
+  "641289076": "AudioBuildInfo",
--- 2018.4.36f1
+++ 2019.1.0a10
@@ -254,0 +255 @@
+  "76251197": "SerializableManagedRefTestClass",
@@ -257,0 +259,4 @@
+  "293259124": "TestObjectWithSpecialLayoutOne",
+  "334799969": "SiblingDerived",
+  "342846651": "TestObjectWithSerializedMapStringNonAlignedStruct",
+  "367388927": "SubDerived",
@@ -259 +263,0 @@
-  "382020655": "PluginBuildInfo",
@@ -261,0 +266,2 @@
+  "478637458": "TestObjectWithSerializedArray",
+  "478637459": "TestObjectWithSerializedAnimationCurve",
@@ -264 +269,0 @@
-  "641289076": "AudioBuildInfo",
@@ -265,0 +271 @@
+  "646504946": "RendererFake",
@@ -272,0 +279 @@
+  "1091556383": "Derived",
@@ -273,0 +281 @@
+  "1154873562": "SceneVisibilityState",
@@ -277,0 +286 @@
+  "1392443030": "TestObjectWithSpecialLayoutTwo",
@@ -281 +290 @@
-  "1571458007": "RenderPassAttachment",
+  "1628831178": "TestObjectVectorPairStringBool",
@@ -284,0 +294 @@
+  "1803986026": "FakeComponent",
@@ -288,0 +299,2 @@
+  "1896753125": "PackageManifest",
+  "1896753126": "PackageManifestImporter",
@@ -290,0 +303,3 @@
+  "1977754360": "NativeObjectType",
+  "1981279845": "TestObjectWithSerializedMapStringBool",
+  "1995898324": "SerializableManagedHost",
--- 2019.1.0a12
+++ 2019.1.0a13
@@ -279,0 +280 @@
+  "1111377672": "PropertyModificationsTargetTestObject",
--- 2019.1.14f1
+++ 2019.2.0a6
@@ -259,0 +260 @@
+  "294290339": "AssemblyDefinitionReferenceImporter",
@@ -271,0 +273 @@
+  "662584278": "AssemblyDefinitionReferenceAsset",
@@ -276,0 +279,3 @@
+  "994735392": "VisualEffectSubgraph",
+  "994735403": "VisualEffectSubgraphOperator",
+  "994735404": "VisualEffectSubgraphBlock",
@@ -280,0 +286 @@
+  "1114811875": "ReferencesArtifactGenerator",
@@ -283,0 +290 @@
+  "1223240404": "MultiArtifactTestImporter",
@@ -309,0 +317 @@
+  "2059678085": "VisualEffectObject",
--- 2019.2.0a6
+++ 2019.2.0a7
@@ -320,2 +320 @@
-  "2089858483": "ScriptedImporter",
-  "2126867596": "TilemapEditorUserSettings"
+  "2089858483": "ScriptedImporter"
--- 2019.2.0a7
+++ 2019.2.0a8
@@ -257,0 +258 @@
+  "277625683": "EmptyObject",
--- 2019.2.12f1
+++ 2019.2.13f1
@@ -265,0 +266 @@
+  "382020655": "PluginBuildInfo",
@@ -271,0 +273 @@
+  "641289076": "AudioBuildInfo",
--- 2019.2.21f1
+++ 2019.3.0a2
@@ -218 +217,0 @@
-  "1044": "MovieImporter",
@@ -266 +264,0 @@
-  "382020655": "PluginBuildInfo",
@@ -273 +270,0 @@
-  "641289076": "AudioBuildInfo",
--- 2019.3.0a3
+++ 2019.3.0a4
@@ -58 +57,0 @@
-  "92": "GUILayer",
@@ -86,3 +84,0 @@
-  "131": "GUITexture",
-  "132": "GUIText",
-  "133": "GUIElement",
--- 2019.3.0a8
+++ 2019.3.0a10
@@ -271,0 +272,2 @@
+  "747330370": "RayTracingShaderImporter",
+  "825902497": "RayTracingShader",
--- 2019.3.0b6
+++ 2019.3.0b7
@@ -260,0 +261 @@
+  "382020655": "PluginBuildInfo",
@@ -266,0 +268 @@
+  "641289076": "AudioBuildInfo",
--- 2019.4.28f1
+++ 2019.4.29f1
@@ -298,0 +299 @@
+  "1736697216": "AndroidAssetPackImporter",
--- 2019.4.37f1
+++ 2019.4.38f1
@@ -296,0 +297 @@
+  "1521398425": "VideoBuildInfo",
--- 2019.4.40f1
+++ 2020.1.0a3
@@ -261 +260,0 @@
-  "382020655": "PluginBuildInfo",
@@ -268 +266,0 @@
-  "641289076": "AudioBuildInfo",
@@ -275,0 +274 @@
+  "850595691": "LightingSettings",
@@ -297 +295,0 @@
-  "1521398425": "VideoBuildInfo",
@@ -300 +297,0 @@
-  "1736697216": "AndroidAssetPackImporter",
--- 2020.1.0a3
+++ 2020.1.0a5
@@ -260,0 +261 @@
+  "382020655": "PluginBuildInfo",
@@ -266,0 +268 @@
+  "641289076": "AudioBuildInfo",
--- 2020.1.0a5
+++ 2020.1.0a7
@@ -251,0 +252 @@
+  "156483287": "ScenesUsingAssets",
--- 2020.1.0a14
+++ 2020.1.0a15
@@ -252,0 +253 @@
+  "171741748": "ArticulationBody",
@@ -267,0 +269 @@
+  "612988286": "SpriteAtlasAsset",
@@ -291,0 +294 @@
+  "1210832254": "SpriteAtlasImporter",
--- 2020.1.0a16
+++ 2020.1.0a17
@@ -280,0 +281 @@
+  "890905787": "VersionControlSettings",
--- 2020.1.18f1
+++ 2020.2.0a7
@@ -196 +195,0 @@
-  "1004": "AssetDatabaseV1",
@@ -268,0 +268 @@
+  "488575907": "ScriptableCamera",
--- 2020.2.0a10
+++ 2020.2.0a11
@@ -268 +267,0 @@
-  "488575907": "ScriptableCamera",
--- 2020.2.0a11
+++ 2020.2.0a12
@@ -282,0 +283 @@
+  "954905827": "RuleSetFileAsset",
@@ -306,0 +308 @@
+  "1777034230": "RuleSetFileImporter",
--- 2020.2.0a15
+++ 2020.2.0a16
@@ -191 +190,0 @@
-  "362": "WorldAnchor",
--- 2020.2.0a17
+++ 2020.2.0a18
@@ -275,0 +276 @@
+  "815301076": "PreviewImporter",
--- 2020.3.14f1
+++ 2020.3.15f1
@@ -304,0 +305 @@
+  "1736697216": "AndroidAssetPackImporter",
--- 2020.3.24f1
+++ 2020.3.25f1
@@ -248 +247,0 @@
-  "76251197": "SerializableManagedRefTestClass",
@@ -253 +251,0 @@
-  "277625683": "EmptyObject",
@@ -255 +252,0 @@
-  "293259124": "TestObjectWithSpecialLayoutOne",
@@ -257,3 +253,0 @@
-  "334799969": "SiblingDerived",
-  "342846651": "TestObjectWithSerializedMapStringNonAlignedStruct",
-  "367388927": "SubDerived",
@@ -264,2 +257,0 @@
-  "478637458": "TestObjectWithSerializedArray",
-  "478637459": "TestObjectWithSerializedAnimationCurve",
@@ -271 +262,0 @@
-  "646504946": "RendererFake",
@@ -289,2 +279,0 @@
-  "1091556383": "Derived",
-  "1111377672": "PropertyModificationsTargetTestObject",
@@ -300 +288,0 @@
-  "1392443030": "TestObjectWithSpecialLayoutTwo",
@@ -304 +291,0 @@
-  "1628831178": "TestObjectVectorPairStringBool",
@@ -310 +296,0 @@
-  "1803986026": "FakeComponent",
@@ -319,3 +304,0 @@
-  "1977754360": "NativeObjectType",
-  "1981279845": "TestObjectWithSerializedMapStringBool",
-  "1995898324": "SerializableManagedHost",
--- 2020.3.32f1
+++ 2020.3.33f1
@@ -290,0 +291 @@
+  "1521398425": "VideoBuildInfo",
--- 2020.3.49f1
+++ 2021.1.0a2
@@ -59 +59 @@
-  "94": "ScriptMapper",
+  "94": "ShaderNameRegistry",
@@ -291 +290,0 @@
-  "1521398425": "VideoBuildInfo",
@@ -293 +291,0 @@
-  "1736697216": "AndroidAssetPackImporter",
--- 2021.1.0a5
+++ 2021.1.0a6
@@ -80 +79,0 @@
-  "125": "HaloLayer",
--- 2021.1.15f1
+++ 2021.1.16f1
@@ -290,0 +291 @@
+  "1736697216": "AndroidAssetPackImporter",
--- 2021.1.29f1
+++ 2021.2.0a5
@@ -291 +290,0 @@
-  "1736697216": "AndroidAssetPackImporter",
--- 2021.2.0a6
+++ 2021.2.0a8
@@ -66 +66 @@
-  "109": "CGProgram",
+  "109": "ShaderInclude",
@@ -309 +309,2 @@
-  "2089858483": "ScriptedImporter"
+  "2089858483": "ScriptedImporter",
+  "2103361453": "ShaderIncludeImporter"
--- 2021.2.0a19
+++ 2021.2.0a20
@@ -270,0 +271 @@
+  "893571522": "CustomCollider2D",
--- 2021.2.0a21
+++ 2021.2.0b1
@@ -254,0 +255 @@
+  "387306366": "MemorySettings",
--- 2021.2.0b3
+++ 2021.2.0b4
@@ -292,0 +293 @@
+  "1736697216": "AndroidAssetPackImporter",
--- 2021.2.13f1
+++ 2021.2.14f1
@@ -291,0 +292 @@
+  "1521398425": "VideoBuildInfo",
--- 2021.3.2f1
+++ 2021.3.3f1
@@ -274,0 +275 @@
+  "947337230": "RoslynAnalyzerConfigAsset",
@@ -293,0 +295,2 @@
+  "1597193336": "RoslynAdditionalFileAsset",
+  "1642787288": "RoslynAdditionalFileImporter",
@@ -304,0 +308 @@
+  "1903396204": "RoslynAnalyzerConfigImporter",
--- 2021.3.29f1
+++ 2021.3.30f1
@@ -293,0 +294 @@
+  "1541671625": "C4DImporter",
--- 2021.3.45f2
+++ 2022.1.0a7
@@ -275 +274,0 @@
-  "947337230": "RoslynAnalyzerConfigAsset",
@@ -293,2 +291,0 @@
-  "1521398425": "VideoBuildInfo",
-  "1541671625": "C4DImporter",
@@ -296,2 +292,0 @@
-  "1597193336": "RoslynAdditionalFileAsset",
-  "1642787288": "RoslynAdditionalFileImporter",
@@ -309 +303,0 @@
-  "1903396204": "RoslynAnalyzerConfigImporter",
--- 2022.1.0a13
+++ 2022.1.0a15
@@ -291,0 +292 @@
+  "1521398425": "VideoBuildInfo",
--- 2022.1.25f1
+++ 2022.2.0a13
@@ -274,0 +275 @@
+  "947337230": "RoslynAnalyzerConfigAsset",
@@ -293,0 +295,4 @@
+  "1557264870": "ShaderContainer",
+  "1597193336": "RoslynAdditionalFileAsset",
+  "1642787288": "RoslynAdditionalFileImporter",
+  "1731078267": "BrokenPrefabAsset",
@@ -304,0 +310 @@
+  "1903396204": "RoslynAnalyzerConfigImporter",
--- 2022.2.0a13
+++ 2022.2.0a16
@@ -245 +245 @@
-  "41386430": "AssetImporterLog",
+  "41386430": "ImportLog",
--- 2022.3.4f1
+++ 2022.3.5f1
@@ -297,0 +298 @@
+  "1660057539": "SceneRoots",
--- 2022.3.7f1
+++ 2022.3.8f1
@@ -293,0 +294 @@
+  "1541671625": "C4DImporter",
--- 2022.3.62f2
+++ 2023.1.0a1
@@ -294 +293,0 @@
-  "1541671625": "C4DImporter",
@@ -299 +297,0 @@
-  "1660057539": "SceneRoots",
--- 2023.1.0a3
+++ 2023.1.0a4
@@ -190,0 +191 @@
+  "900": "MarshallingTestObject",
--- 2023.1.0a15
+++ 2023.1.0a16
@@ -256,0 +257,2 @@
+  "403037116": "BuildMetaDataImporter",
+  "403037117": "BuildInstructionImporter",
@@ -267,0 +270 @@
+  "780535461": "BuildArchiveImporter",
--- 2023.1.0b17
+++ 2023.1.0b18
@@ -268,0 +269 @@
+  "702665669": "DifferentMarshallingTestObject",
--- 2023.1.3f1
+++ 2023.1.4f1
@@ -302,0 +303 @@
+  "1660057539": "SceneRoots",
--- 2023.1.8f1
+++ 2023.1.9f1
@@ -298,0 +299 @@
+  "1541671625": "C4DImporter",
--- 2023.1.22f1
+++ 2023.2.0a6
@@ -269 +268,0 @@
-  "702665669": "DifferentMarshallingTestObject",
@@ -299 +297,0 @@
-  "1541671625": "C4DImporter",
@@ -304 +301,0 @@
-  "1660057539": "SceneRoots",
--- 2023.2.0a8
+++ 2023.2.0a9
@@ -253,0 +254 @@
+  "355983997": "AudioResource",
@@ -291,0 +293 @@
+  "1233149941": "AudioContainerElement",
@@ -292,0 +295 @@
+  "1307931743": "AudioRandomContainer",
--- 2023.2.0a14
+++ 2023.2.0a15
@@ -269,0 +270 @@
+  "702665669": "DifferentMarshallingTestObject",
@@ -317,0 +319 @@
+  "1931382933": "UIRenderer",
--- 2023.2.0a18
+++ 2023.2.0a19
@@ -266,0 +267 @@
+  "655991488": "MultiplayerManager",
@@ -305,0 +307,2 @@
+  "1652712579": "MultiplayerRolesData",
+  "1660057539": "SceneRoots",
--- 2023.2.0b3
+++ 2023.2.0b4
@@ -302,0 +303 @@
+  "1541671625": "C4DImporter",
--- 2023.3.0a8
+++ 2023.3.0a10
@@ -84 +84 @@
-  "134": "PhysicMaterial",
+  "134": "PhysicsMaterial",
--- 6000.0.0b13
+++ 6000.0.0b15
@@ -246,0 +247 @@
+  "55640938": "GraphicsStateCollection",
--- 6000.1.0a4
+++ 6000.1.0a5
@@ -312,0 +313 @@
+  "1740304944": "VulkanDeviceFilterLists",
--- 6000.2.6f2
+++ 6000.2.7f2
@@ -317,0 +318 @@
+  "1798438448": "D3D12DeviceFilterLists",
--- 6000.2.9f1
+++ 6000.3.0a2
@@ -271,0 +272 @@
+  "687930938": "BlobObject",
@@ -318 +318,0 @@
-  "1798438448": "D3D12DeviceFilterLists",
--- 6000.3.0a3
+++ 6000.3.0a5
@@ -291,0 +292 @@
+  "1120581460": "RenderAs2D",
@@ -305 +305,0 @@
-  "1541671625": "C4DImporter",
@@ -318,0 +319 @@
+  "1798438448": "D3D12DeviceFilterLists",
--- 6000.3.0b7
+++ 6000.4.0a2
@@ -254,0 +255 @@
+  "330615474": "ContentSummary",
@@ -275 +275,0 @@
-  "780535461": "BuildArchiveImporter",
@@ -291 +291,3 @@
-  "1114811875": "ReferencesArtifactGenerator",
+  "1096314112": "BlockShaderSyntaxTree",
+  "1096314113": "BlockShaderContainer",
+  "1096314114": "BlockShaderErrors",
@@ -292,0 +295 @@
+  "1149764431": "BlockShaderSourceArtifact",
@@ -307 +309,0 @@
-  "1557264870": "ShaderContainer",
@@ -336,0 +339 @@
+  "2097406814": "BlockShaderImporter",
